### DISCUSSION TITLE
第20期：AI是专业能力的放大器 | 信息差——独立开发者出海周刊

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/gapis.money#39] [General]
# 第20期：AI是专业能力的放大器 | 信息差——独立开发者出海周刊

Knowledge is power, info-gap is money!「信息差——独立开发者出海周刊」是一个帮助独立开发者缩小信息差的技术周刊。

https://gapis.money/weekly/2024-09-30_020

**Comment by WymanY** [comment]
兄弟，怎么有一阵子没更新了，不是周报嘛 🤦‍♂️

**Reply by weijunext** [reply]
上周碰上国庆，明天继续更
